geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalLocatorIntegrationTest.java

```diff
@@ -97,16 +118,16 @@ public void constructs() {
 
     assertThatCode(() -> {
       internalLocator =
-          new InternalLocator(port, loggingSession, logFile, logWriter, securityLogWriter,
+          new InternalLocator(port, loggingSession, logFile, null, null,
```

Comment:
My read of `InternalLocator` is that it's ok to provide `null` log writers to the constructor ✓